waltz-ng/client/data-types/routes.js

```diff
@@ -0,0 +1,42 @@
+const baseState = {
+    resolve: {
+        dataTypes: [
+            'DataTypesService',
+            (dataTypesService) => dataTypesService.loadDataTypes()
+        ]
+    }
+};
+
+
+const listState = {
+    url: 'data-types',
+    views: {'content@': require('./data-type-list') }
+};
+
+
+const viewByCodeState = {
+    url: 'data-types/code/{code}',
+    views: {'content@': require('./data-type-view') },
+    redirectTo: 'main.data-type.view',
```

Comment:
I need to be able to access the state parameter, as long as i could do that
it should be fine.
I'll make changes for the other suggestions you made too.

I'll just finish off the list page, check it in and then work on the
feedback

On Thu, Aug 11, 2016 at 5:53 PM, David Watkins <notifications@github.com>
wrote:

> In waltz-ng/client/data-types/routes.js
> <https://github.com/khartec/waltz/pull/417#discussion_r74459910>:
>
> > +            (dataTypesService) => dataTypesService.loadDataTypes()
> > +        ]
> > +    }
> > +};
> > +
> > +
> > +const listState = {
> > +    url: 'data-types',
> > +    views: {'content@': require('./data-type-list') }
> > +};
> > +
> > +
> > +const viewByCodeState = {
> > +    url: 'data-types/code/{code}',
> > +    views: {'content@': require('./data-type-view') },
> > +    redirectTo: 'main.data-type.view',
>
> ok - I'm probably forgetting/overlooking something crucial ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/417/files/e4bc887bdadc42f03d5e44e0876776ce30ea1a5c#r74459910>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC452yswZj1oFgu0pASZtqE6FcbPx89Hks5qe1ODgaJpZM4JiBOI>
> .
>



-- 


*Kamran Saleem*
t  +44 (0) 7732 078654
e  kamran@thinkincode.co.uk
w thinkincode.co.uk